docs/static/releasenotes.asciidoc

```diff
@@ -33,6 +34,29 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-9-2]]
+=== Logstash 7.9.2 Release Notes
+
+==== Notable issues fixed
+
+===== Secret store thread safety issues with multiple pipelines
+
+Since `7.8.0`, a change to optimise the speed of loading variables from the Logstash Secret Store could cause Logstash not to be able to start when the feature was used in conjunction with multiple pipelines. This has now been fixed, and you can read the details here: https://github.com/elastic/logstash/pull/12236[#12236]
+
+===== App Search output startup failure
+
+Since `7.9.0`, a regression was introduced which prevented pipelines using the Elastic App Search output from starting. This release fixes support for this plugin, you can read the details here: https://github.com/logstash-plugins/logstash-output-elastic_app_search/pull/18[#18], https://github.com/elastic/logstash/pull/12251[#12251]
+
```

Comment:
```suggestion
[[jdk15-compat]]
==== Compatibility notice: {ls} and JDK 15

{ls} is not yet compatible with JDK 15.

While we are working to support JDK 15, we encourage you to use supported JDK
versions (8, 11 or 14). See <<ls-jvm>> for details and the
https://www.elastic.co/support/matrix#matrix_jvm[Elastic Support Matrix] for the
official word on supported versions across products and releases.
```

Reply:
I'm good with this

Reply:
Cool.  I'll do the final review after you accept the change or implement it on your own (no preference).  I'd like to have a clean PR preview because making github suggestions can be tricky wrt asciidoc formatting.

Reply:
I committed your suggestion... and promptly broke the formatting, should be fixed now